cmd/rook/rook/rook.go

```diff
@@ -69,6 +69,7 @@ var (
 //  2. environment variables (upper case, replace - with _, and rook prefix. For example, discovery-url is ROOK_DISCOVERY_URL)
 //  3. command line parameter
 func init() {
+	RootCmd.CompletionOptions.DisableDefaultCmd = true
```

Comment:
The auto completion feature can be handy. I suggest we don't remove it.

Reply:
Auto-completion will be useful when we are generating auto-completion and putting it in right place for the shell to consume that. Also, it is more useful for cli-based tool. IMO we don't use rook cli much to have a completion but I'm fine with not removing if we agree.

Reply:
I'm ok with either way. Agree that we don't use this rook binary directly and won't need this auto completion. Lets confirm with @BlaineEXE as he has added these changes.

Reply:
We do actually use the binary directly for the multus validation tool.

Users download the tool and run it locally, so in this case, the rook binary is used as a local CLI. Let's keep this for now.

Reply:
This line should be a hint that the completion command was intentionally enabled and not accidentally left enabled by oversight: 

https://github.com/rook/rook/blob/b3dd74ea20e740c2b7ce760acd93716ac8374cfe/cmd/rook/rook/rook.go#L75